### PR TITLE
Sdcch leak

### PR DESCRIPTION
**Pull request by kheimerl**

### CODE REVIEW

**Comment by kheimerl** [comment]
ptal

File: GSM/GSML2LAPDm.cpp [hunk 1]
```diff
@@ -762,14 +762,12 @@ void L2LAPDm::receiveUFrameDM(const L2Frame& frame)
...
 	switch (mState) {
-	case LAPDStateUnused:
-	case LinkReleased:
-		// GSM 04.06 5.4.5: In idle state all "other" frame types ignored.
-		return;
 	case AwaitingRelease:
 		// GSM 4.06 5.4.4.2 
 		releaseLink(true,L3_RELEASE_CONFIRM);
 		return;
+	case LAPDStateUnused:
```
Comment (kkroo) [inline GSM/GSML2LAPDm.cpp]: does the idle state still work with this change?

Comment (kheimerl) [inline GSM/GSML2LAPDm.cpp]: it should, the code path wasn't supposed to be used anyhow.

**Comment by kkroo** [comment]
SDDCH load appears to look good with the fix. LGTM

**Pull request merged**